src/test/java/com/softserveinc/ita/CheckoutPageTest.java

```diff
@@ -0,0 +1,66 @@
+package com.softserveinc.ita;
+
+import com.softserveinc.ita.pageobjects.CheckoutPage;
+import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.ProductPage;
+import com.softserveinc.ita.utils.runners.TestRunner;
+import org.testng.annotations.Test;
+
+import static com.softserveinc.ita.repos.CustomerOrderDataRepo.getCustomerOrderData;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CheckoutPageTest extends TestRunner {
+
+    protected HomePage homePage = new HomePage();
```

Comment:
leave only home page variable as class var, make it private final

others should be inited in a test near the places where they being needed or get they through methods return statements